packages/ec-core/src/individual/ec.rs

```diff
@@ -111,20 +110,22 @@ impl<GG, S> WithScorer<S> for GG {
 // GG is Genome generator
 // S is Scorer
 // R is the TestResult type
```

Comment:
Yea let's remove the comments. The `...T` was more to avoid name collisions with the corresponding traits. More generally I would say we should just decide on a naming convention for generics in this project, whatever it is in the end (short names, ending them with a `T`, trying to find another descriptive synonym, ...) although I favor something that is more descriptive, so that comments like the ones above are not needed

Reply:
See discord for more context